CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Bugs fixed
 
 * Don't count required keyword args when specifying `CountKeywordArgs: false` for `ParameterLists`. ([@sumeet][])
+* `Sample` no longer registers `#first` with a block as an offence. ([@chastell][])
```

Comment:
@jonas054 Fixed, as I sadly note yet another instance of the awful American cultural imperialism. ;)